legistar/events.py

```diff
@@ -195,17 +195,21 @@ def _scrapeWebCalendar(self):
         web_info = {}
 
         for event, _ in web_scraper.events(follow_links=False):
-            # Make the dict key (name, datetime.datetime), and add it.
-            response = web_scraper.get(event['iCalendar']['url'], verify=False)
-            event_time = web_scraper.ical(response.text).subcomponents[0]['DTSTART'].dt
-            event_time = pytz.timezone(self.TIMEZONE).localize(event_time)
-
-            key = (event['Name']['label'],
-                   event_time)
+            key = self._event_key(event, web_scraper)
             web_info[key] = event
 
         return web_info
 
+    def _event_key(self, event, web_scraper):
+        response = web_scraper.get(event['iCalendar']['url'], verify=False)
```

Comment:
Can you add a comment or docstring that states the purpose of this method is to generate the `tuple(event name, event time)`  key under which the event is stored in the `web_info` dict? Hard to grok on first read until you see what's returned.